scripts/2.TCGA-MLexample-covariates.py

```diff
@@ -234,7 +237,7 @@ def get_threshold_metrics(y_true, y_pred):
 }
 
 
-# In[20]:
+# In[17]:
```

Comment:
Surprising to me that the `full`  (covariates + expression) performs almost equivalently to `expression` for TP53. Good to know. CC @gwaygenomics